brion/morphologyPlugin.h

```diff
@@ -40,31 +37,28 @@ class MorphologyInitData : public PluginInitData
 public:
     explicit MorphologyInitData(const URI& uri)
         : PluginInitData(uri, MODE_READ)
-        , _version(MORPHOLOGY_VERSION_H5_1_1)
-        , _family(FAMILY_NEURON)
+        , version(MORPHOLOGY_VERSION_H5_1_1)
+        , family(FAMILY_NEURON)
     {
     }
 
-    MorphologyInitData(const URI& uri, const MorphologyVersion version,
+    MorphologyInitData(const URI& uri, const MorphologyVersion v,
                        const unsigned int accessMode)
         : PluginInitData(uri, accessMode)
-        , _version(version)
-        , _family(FAMILY_NEURON)
+        , version(v)
+        , family(FAMILY_NEURON)
     {
     }
 
-    MorphologyInitData(const URI& uri, const CellFamily family)
+    MorphologyInitData(const URI& uri, const CellFamily f)
         : PluginInitData(uri, MODE_WRITE)
-        , _version(MORPHOLOGY_VERSION_H5_1_1)
-        , _family(family)
+        , version(MORPHOLOGY_VERSION_H5_1_1)
+        , family(f)
     {
     }
 
-    MorphologyVersion getVersion() const { return _version; }
-    CellFamily getFamily() const { return _family; }
-protected:
-    const MorphologyVersion _version;
-    const CellFamily _family;
+    MorphologyVersion version;
+    CellFamily family;
```

Comment:
Not needed if you just keep the family and version in the morphology plugin class instead of a copy of this object.